src/backend/distributed/planner/multi_router_planner.c

```diff
@@ -830,7 +830,7 @@ IsTableLocallyAccessible(Oid relationId)
 	ShardInterval *shardInterval = linitial(shardIntervalList);
 	uint64 shardId = shardInterval->shardId;
 	ShardPlacement *localShardPlacement =
-		ShardPlacementOnGroup(shardId, GetLocalGroupId());
+		ActiveShardPlacementOnGroup(GetLocalGroupId(), shardId);
```

Comment:
This change shouldn't really matter, because this function is only called for citus local tables. Which cannot have inactive shard placements.